storybook/src/pages/amsterdam/article/ArticleBody.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+  AspectRatio,
+  Column,
+  Grid,
+  Heading,
+  Image,
+  Overlap,
+  Paragraph,
+  Spotlight,
+} from '@amsterdam/design-system-react'
+import { exampleHeading, exampleParagraph } from '../../../components/shared/exampleContent'
+
+export type ArticleContentProps = {}
+
+export const ArticleBody = () => (
+  <>
+    <Grid paddingVertical="large">
+      <Grid.Cell span={{ narrow: 4, medium: 8, wide: 10 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
+        <Column gap="medium">
+          <Paragraph size="large">{exampleParagraph()}</Paragraph>
+          <Paragraph>{exampleParagraph()}</Paragraph>
+          <Heading level={2}>{exampleHeading()}</Heading>
+          <Paragraph>{exampleParagraph()}</Paragraph>
```

Comment:
It's a bit of a hallucinatory article now ;) Do we want to keep using random content here?